easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
+builddependencies = [(python, pyver)]
```

Comment:
@boegel  I just want to be sure that changing the system `Python` will not influence the usage of `xcb-proto` (it makes .pyc and .pyo files as well).